src/CMakeLists.txt

```
@@ -0,0 +1,79 @@
+add_subdirectory(vmec_interface)
+
+
+add_executable(stella.x  ${SOURCES} )
```

Don't worry about the suffix -- CMake will make sure that it has e.g. `.exe` on Windows, and no suffix (not required) on Linux

```suggestion
add_executable(stella ${SOURCES} )
```

All the subsequent commands will need updating too